src/objects/Customizable_Rollup_Settings__c.object

```diff
@@ -135,7 +135,7 @@
         <description>Only used by the new Customizable Rollups engine</description>
         <externalId>false</externalId>
         <inlineHelpText>The maximum number of related Opportunities on an Account or Contact allowed in the non-Skew Mode Rollup Batch Jobs. An Account or Contact with more than this number of related Opportunities will always be rolled up using Skew Mode.</inlineHelpText>
-        <label>Rollups: Max Related Record in Skew Mode</label>
+        <label>Skew Mode Threshold</label>
```

Comment:
spelling error

Reply:
"Threshold" is correct. Is there something else I'm not seeing?

Reply:
Sorry. Need to not review code at the end of a long Friday.